asset/css/markbind.css

```diff
@@ -16,3 +16,15 @@ blockquote {
         background-color: #fff;
     }
 }
+
+.radio-list-item,
+.task-list-item {
+    list-style-type: none;
+    margin: 0 0 0 -1em;
+}
+
+.radio-list-input,
+.task-list-item-checkbox {
+    height: 10px;
+    width: 10px;
+}
```

Comment:
Why is this block necessary?

Reply:
I added so that the text will be inline regardless of list type (radio, task, bullet).

If i remove it the example above will be:
![image](https://user-images.githubusercontent.com/22221132/37241914-fa0d2a24-249b-11e8-8a71-93aa1963ad15.png)

Reply:
This reduces the size of a click target, which is undesirable.

Since we are already tweaking `margin` above, how about using `-1.2em` instead of `-1em`?

![task, bullet, radio](https://user-images.githubusercontent.com/14091939/37242881-59ea90c6-24ab-11e8-8e8d-3601caf9b5fe.png)

Note that both ways are browser-dependent, so we shouldn't fret over this. If you do this:

```css
html, body {
    font-size: unset;
}
```

![task, bullet, radio + unset](https://user-images.githubusercontent.com/14091939/37243139-d57d4298-24af-11e8-9178-41ae9f96e16d.png)

If you change the font size of your browser, the misalignment is even more apparent.

Even GitHub doesn't care:

- [ ] task
- bullet

Reply:
> Since we are already tweaking margin above, how about using -1.2em instead of -1em?

👍 Here's what it looks like (original). On its right is something i want to suggest which is adding
```css
.radio-list-input,
.task-list-item-checkbox {
    vertical-align: text-bottom;
}
```
It makes the input more aligned and the lists become more compact too.

![image](https://user-images.githubusercontent.com/22221132/37243417-bd277cfa-24b3-11e8-8993-90522bc8c42a.png)

Reply:
That misaligns the input on Safari 🤣

original | vertical-align: text-bottom;
-- | --
<img width="65" alt="screenshot 2018-03-10 22 52 48" src="https://user-images.githubusercontent.com/14091939/37243539-15ad1202-24b6-11e8-851f-87128d652155.png"> | <img width="65" alt="screenshot 2018-03-10 22 53 34" src="https://user-images.githubusercontent.com/14091939/37243540-1c8c1c26-24b6-11e8-98c3-ecf1d8779dd1.png">